Makefile

```diff
@@ -113,7 +146,7 @@ GH_REPO_NAME ?= rancher-turtles
 GH_REPO ?= $(GH_ORG_NAME)/$(GH_REPO_NAME)
 
 # Allow overriding the imagePullPolicy
-PULL_POLICY ?= Always
+PULL_POLICY ?= IfNotPresent
```

Comment:
why was this changed?

Reply:
This causes the cluster to pull the tag from outside. This should not be happening, e2e run should use the latest local image if it is specified. And it always is for rancher-turtles